.github/workflows/e2e.yml

```
@@ -59,15 +59,13 @@
       - name: Run tests
-        run: pnpm exec playwright test
-        working-directory: e2e
+        run: pnpm run test:e2e
 
       - uses: actions/upload-artifact@v4
         if: always()
         with:
           name: playwright-report
-          path: e2e/playwright-report/
+          path: playwright-report/
           retention-days: 30
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)